src/legacy/core_plugins/expressions/public/np_ready/public/loader.ts

```diff
@@ -101,19 +121,19 @@ export class ExpressionLoader {
 
   private execute = async (
```

Comment:
This is somewhat unrelated but it's confusing to have a method named `execute` and a function called `execute` that is being used by the method. I would suggest renaming `execute` to `loadData` or similar

Reply:
i see you are addressing this in your PR so i will leave it out of this one.